templates/taxi/manufacturer_list.html

```diff
@@ -8,6 +9,12 @@ <h1>
     </a>
   </h1>
 
+<form class="form-inline" method="get">
+    {% csrf_token %}
```

Comment:
The CSRF token is not needed in this form since you are using the GET method. CSRF protection is important when dealing with forms that have POST, PUT, PATCH, DELETE methods. You can safely remove the line with the csrf_token.